src/main/java/net/sf/jabref/gui/desktop/os/Windows.java

```diff
@@ -3,24 +3,38 @@
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Paths;
+import java.util.Optional;
+
+import net.sf.jabref.external.ExternalFileType;
+import net.sf.jabref.external.ExternalFileTypes;
 
 public class Windows implements NativeDesktop {
+
+    private static String DEFAULT_EXECUTABLE_EXTENSION = ".exe";
+
+
+    @Override
+    public void openFile(String filePath, String fileType) throws IOException {
+        Optional<ExternalFileType> type = ExternalFileTypes.getInstance().getExternalFileTypeByExt(fileType);
+        if (type.isPresent() && !type.get().getOpenWithApplication().isEmpty()) {
```

Comment:
The functionality "if recognized type, then open with special application" should be added to all OS in a consistent way. I would propose to implement a corresponding method in `NativeDesktop`, which calls a new method `openFileDirectly` (or better named). This new method is then overwritten by the specific OS implementations.